### ISSUE TITLE
Onboarding Philip Smith

### ISSUE THREAD

**Issue by ryeleo** [routeviews/issues#29]
### Desired Username

pfs

### SSH Key

```shell
Already done
```

### New Maintainer Initial Tasks

- [X] Share Public SSH key (RSA, >2048 bytes)
- [ ] Review the [official "GitHub Flow" documentation](https://docs.github.com/en/get-started/quickstart/github-flow).
- [ ] (Optional) If  is new to you, consider reviewing and bookmarking the ["Git Guides" provided by GitHub](https://github.com/git-guides).

### Existing Maintainer Tasks

- [X] (UO Staff only) Add to [routeviews Active Directory Security Group](https://portal.azure.com/#view/Microsoft_AAD_IAM/GroupDetailsMenuBlade/~/Overview/groupId/9e7d2b47-7eb1-476c-818c-b7ba065c4e71). (This enables access to UO VMs)
- [X] Add to ["Route Views Maintainers" GitHub Team](https://github.com/orgs/routeviews/teams/maintainers/members?add=true).
- [x] Deploy public SSH Key to RV infrastructure (via Ansible). Follow the [New Route Views Maintainer workflow (TODO)](https://github.com/routeviews/infra/blob/master/docs/contribute/new_rv_maintainer.md).
- [X] Setup <username>@routeviews.org email account.
- [X] Provide access to [our Grafana instance](http://rvmonitor.routeviews.org:3000/).
- [ ] Admit/ensure access to RV in PeeringDB.
- [X] Ensure access to RV Slack.
- [X] Provide access to the private Maintainer-only Slack channels. Today this includes: "peer requests" and "infrastructure".
- [X] Share Route Views KeePass Master Password. (e.g., do this via RV Slack using [the "Secret Message" app](https://secretmessage.xyz/))
- [ ] Help the new member complete their first 'peer request'! (e.g., schedule a time within the next work-week)

### New Maintainer Final Tasks

- [X] Set up your mail client with your new Route Views email address. Refer to our [mail configuration documentation](https://github.com/routeviews/infra/blob/master/docs/contribute/mail_settings.md)
- [x] (BONUS) Set up the helpmail@routeviews.org email address also, giving you access to vast history of emails (credentials in [RV KeePass](https://github.com/routeviews/infra/blob/master/RouteViews.kdbx)).
- [x] Ensure access to [Route Views Slack](https://routeviewsworkspace.slack.com).
- [ ] Review the [RV Maintainer Handbook (TODO update link to GH Pages)](https://github.com/routeviews/handbook).
- [x] Set up a PeeringDB account, and ['request Affiliation' with Route Views  (ASN 6447) on your PeeringDB profile page](https://www.peeringdb.com/profile)
- [x] Ensure login, and take a look through dashboards in [our Grafana instance](http://rvmonitor.routeviews.org:3000/).
- [ ] Complete your first peer request! 😁 (with help from an existing maintainer)

**Comment by ryeleo** [comment]
Blocked by a bug in the peer request automation: https://github.com/routeviews/infra/issues/70

**Comment by ryeleo** [comment]
Duplicate of https://github.com/routeviews/infra/issues/118
